test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
I agree with @d2lam. Even though it's not validated/necessary, it is nice (for someone less familiar with what is expected) to see sample data in the tests that are closer to the real thing than something arbitrary. 
It doesn't have to be a valid UUID, but having just the curly braces and a number (`{1234}`) seems closer to a UUID than `wyattLib`.